index.js

```diff
@@ -0,0 +1,68 @@
+const readline = require('readline-sync');
+const parse = require('csv-parse/lib/sync');
+const log4js = require('log4js');
+const logger = log4js.getLogger('logs\\debug.log');
+const fs = require('fs');
+const splitPoundsPennies = require('./splitPoundsPennies');
+const createAccountArray = require('./createAccountArray');
+const addSubtractInAccounts = require('./addSubtractInAccounts');
+const parseIntTransacAmounts = require('./parseIntTransacAmounts');
+const recombinePenniesPounds = require('./recombinePenniesPounds');
+const userRequestList = require('./userRequestList');
+
+log4js.configure({
+    appenders: {
+        file: { type: 'fileSync', filename: 'logs\\debug.log' }
+    },
+    categories: {
+        default: { appenders: ['file'], level: 'debug'}
+    }
+});
+
+class Transaction {
+    constructor(Date,FromAccount,ToAccount,Narrative,Amount,amountPounds,amountPennies){
+    this.Date=Date;
+    this.FromAccount=FromAccount;
+    this.ToAccount=ToAccount;
+    this.Narrative=Narrative;
+    this.Amount=Amount;
+    this.amountPounds=amountPounds;
+    this.amountPennies=amountPennies;
+    }
+}
+function runMyProgram() {
+    console.log('\nPlease enter name of file to be used:');
+    const fileChoice=readline.prompt();
+    fileType=fileChoice.split(".")[1];
+
+    if (fileType==='csv') {

```

Comment:
You can use a `switch` rather than an `if` here:

    switch (fileType) {
      case 'csv':
         ...
         break; // because of the slightly odd syntax where cases don't have {} blocks, you need to break at the end of each case
      case 'json':
        const fileContents=...;
        var allTransactions=...;
        break;
      default:
        console.log(...);
        runMyProgram();
        break;
    }